uint8 target_id
uint8 ARUCO_ID_0 = 0
uint8 ARUCO_ID_1 = 1
uint8 ARUCO_ID_2 = 2
uint8 ARUCO_ID_3 = 3
uint8 ARUCO_ID_4 = 4
uint8 OBJ_MALLET = 5
uint8 OBJ_BOTTLE = 6
uint8 ANY = 9
---
---
